src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -2106,7 +2173,7 @@ def process_declarations(self, parent, nodes, arg_list,
         :type nodes: list of :py:class:`fparser.two.utils.Base`
         :param arg_list: fparser2 AST node containing the argument list.
         :type arg_list: :py:class:`fparser.Fortran2003.Dummy_Arg_List`
-        :param visibility_map: mapping of symbol names to explicit
+        :param visibility_map: mapping of symbol names to explicit \
                         visibilities.
```

Comment:
I has told we are now removing this backslashes as they are rendered properly in the docs if the next line is indented.

Reply:
Removed.